### ISSUE TITLE
[BUG] MySQL syntax error in tad\WPBrowser\dropWpTables near 'IF EXISTS'

### ISSUE THREAD

**Issue by EvanShaw** [lucatume/wp-browser#567]
**Environment**
OS: Linux
PHP version: 7.4
Installed Codeception version: 4.1.31
Installed wp-browser version: 3.1.4
WordPress version: 5.9.2
Local development environment: Docker
WordPress structure and management: default

**Codeception configuration file**  
```yaml
paths:
    tests: tests
    output: tests/_output
    data: tests/_data
    support: tests/_support
    envs: tests/_envs
actor_suffix: Tester
extensions:
    enabled:
        - Codeception\Extension\RunFailed
    commands:
        - Codeception\Command\GenerateWPUnit
        - Codeception\Command\GenerateWPRestApi
        - Codeception\Command\GenerateWPRestController
        - Codeception\Command\GenerateWPRestPostTypeController
        - Codeception\Command\GenerateWPAjax
        - Codeception\Command\GenerateWPCanonical
        - Codeception\Command\GenerateWPXMLRPC
params:
    - .env.testing
```

**Suite configuration file**
```yaml
# Codeception Test Suite Configuration
#
# Suite for unit or integration tests that require WordPress functions and classes.

actor: WpunitTester
modules:
    enabled:
        - WPLoader
        - Welcart
        - \Helper\Wpunit
        - \Helper\Devenv
        - \Helper\TransactionSampleRequests
    config:
        WPLoader:
            wpRootFolder: "%WP_ROOT_FOLDER%"
            dbName: "%TEST_DB_NAME%"
            dbHost: "%TEST_DB_HOST%"
            dbUser: "%TEST_DB_USER%"
            dbPassword: "%TEST_DB_PASSWORD%"
            tablePrefix: "%TEST_TABLE_PREFIX%"
            domain: "%TEST_SITE_WP_DOMAIN%"
            adminEmail: "%TEST_SITE_ADMIN_EMAIL%"
            language: "ja"
            wpDebug: false
            installationTableHandling: drop
            title: "Test"        
            plugins: ["usc-e-shop/usc-e-shop.php", "wcex_atodene/wcex_atodene.php"]
            activatePlugins: ["usc-e-shop/usc-e-shop.php", "wcex_atodene/wcex_atodene.php"]
```

**Describe the bug**
When the option `installationTableHandling` for the WPLoader module is set to `drop`, a syntax error occurs when wp-browser attempts to drop tables.

**Output**
```
WordPress database error You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'IF EXISTS' at line 1 for query DROP TABLE wp_users IF EXISTS made by tad\WPBrowser\dropWpTables

In bootstrap.php line 150:
                                                                               
  thrown in /app/wp-content/plugins/wcex_atodene/vendor/lucatume/wp-browser/s  
  rc/tad/WPBrowser/wp.php on line 28   
```

**To Reproduce**
Set `installationTableHandling` to `drop` and run any test

**Expected behavior**
No syntax error

**Additional context**
The error occurs because of the following line:
```php
$dropped = $wpdb->query("DROP TABLE {$prefixedTable} IF EXISTS");
```
As far as I know, the proper syntax for mysql is this:
```php
$dropped = $wpdb->query("DROP TABLE IF EXISTS {$prefixedTable}");
```
Using the above syntax fixed the error for me.

**Comment by lucatume** [comment]
Thanks @EvanShaw for reporting the issue and taking the time to fill in all the information.
I will look into this and post updates on the issue.

**Comment by lucatume** [comment]
Fixed in [version `3.1.5`](https://github.com/lucatume/wp-browser/releases/tag/3.1.5).
